plugins/module_utils/identity/keycloak/keycloak.py

```diff
@@ -55,13 +55,14 @@ def keycloak_argument_spec():
     :return: argument_spec dict
     """
     return dict(
-        auth_keycloak_url=dict(type='str', aliases=['url'], required=True, no_log=False),
+        auth_keycloak_url=dict(type='str', aliases=['url'], required=True),
```

Comment:
```suggestion
        auth_keycloak_url=dict(type='str', aliases=['url'], required=True, no_log=False),
```
Otherwise sanity checks are fail.